tests/e2e/lib.sh

```diff
@@ -17,11 +17,41 @@ deploy_stackrox() {
     wait_for_api
     setup_client_TLS_certs
 
-    SENSOR_IMAGE_TAG=${SENSOR_IMAGE_TAG:-$MAIN_IMAGE_TAG}
-    if [ "$SENSOR_IMAGE_TAG" != "$MAIN_IMAGE_TAG" ]; then
-        echo "Deploying sensor with custom image: $SENSOR_IMAGE_TAG"
+    deploy_sensor
+    echo "Sensor deployed. Waiting for sensor to be up"
+    sensor_wait
+
+    # Bounce collectors to avoid restarts on initial module pull
+    kubectl -n stackrox delete pod -l app=collector --grace-period=0
+
+    sensor_wait
+
+    touch "${STATE_DEPLOYED}"
+}
+
+deploy_stackrox_with_custom_sensor() {
+    if [ -z $1 ]; then
```

Comment:
This'll blow up due to set -u and it being an unbound variable. This is used elsewhere in the file:
```suggestion
    if [[ "$#" -ne 1 ]]; then
```

Reply:
Done